apps/api/src/deployment/routes/deployments/deployments.router.ts

```diff
@@ -0,0 +1,71 @@
+import { createRoute } from "@hono/zod-openapi";
+import { container } from "tsyringe";
+import { z } from "zod";
+
+import { OpenApiHonoHandler } from "@src/core/services/open-api-hono-handler/open-api-hono-handler";
+import { DeploymentController } from "@src/deployment/controllers/deployment/deployment.controller";
+import { CreateDeploymentRequestSchema, CreateDeploymentResponseSchema, GetDeploymentParamsSchema, GetDeploymentResponseSchema } from "@src/deployment/http-schemas/deployment.schema";
+
+const getRoute = createRoute({
+  method: "get",
+  path: "/v1/deployments/{dseq}",
```

Comment:
`dseq` is not a unique identifier for a deployment. `userId` + `dseq` or `address/owner` + `dseq` is. In our case user seems to be more appropriate. So the interface should prob be something like:

```suggestion
  path: "/v1/deployments/${userId}/{dseq}",
```

It's not perfect so it's totally open to discussion, however user should be taken into account

Reply:
How about if it had the same interface as `GET /v1/bids`, with `?dseq&userId`, userId being optional as it can be derived from auth, but also can be forced when you're an admin?
Maybe it should be `GET /v1/x/{dseq}?userId` now that I think about it. What do you think?

Reply:
Ok, changed it so now it is working the same as `GET /v1/bids` (both params in query string).